executor/adapter.go

```diff
@@ -1093,6 +1093,7 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, hasMoreResults bool) {
 		trace.Log(a.GoCtx, "details", slowLog)
 	}
 	logutil.SlowQueryLogger.Warn(slowLog)
+	logutil.BgLogger().Warn("SLOW_QUERY", zap.String("details", slowLog))
```

Comment:
Can we remove the `logutil.SlowQueryLogger`?

Reply:
I don't think so. I'll add a global variable to control if it should be printed in one line or not.